# DP: remove blanks from target_bit, see https://bugs.debian.org/814978

TODO: What problem does this patch solve?
TODO: A fix in impbit.adb would be more readable and efficient.
TODO: Forward to upstream bugzilla and add a reference here.

--- a/src/gcc/testsuite/ada/acats/run_all.sh
+++ b/src/gcc/testsuite/ada/acats/run_all.sh
@@ -130,7 +130,7 @@ if [ $? -ne 0 ]; then
    exit 1
 fi
 target_run $dir/support/impbit > $dir/support/impbit.out 2>&1
-target_bit=`cat $dir/support/impbit.out`
+target_bit=`sed -e 's/[ \r]//g' $dir/support/impbit.out`
 echo target_bit="$target_bit" >> $dir/acats.log
 
 # Find out a suitable asm statement
